x-pack/plugin/logstash/src/main/java/org/elasticsearch/xpack/logstash/Logstash.java

```diff
@@ -58,7 +60,8 @@ boolean isTransportClient() {
 
     public UnaryOperator<Map<String, IndexTemplateMetaData>> getIndexTemplateMetaDataUpgrader() {
         return templates -> {
-            TemplateUtils.loadTemplateIntoMap("/" + LOGSTASH_TEMPLATE_NAME + ".json", templates, LOGSTASH_TEMPLATE_NAME,
+            templates.keySet().removeIf(OLD_LOGSTASH_INDEX_NAME::equals);
```

Comment:
This might be a bad idea because I could easily see someone creating a template named this, but I'm not sure if we want to leave it hanging around either.